viz/cmd/testdata/tap_busy_output_wide.golden

```diff
@@ -1,2 +1,2 @@
-req id=1:0 proxy=out src=0.0.0.1:0 dst=[ff01::1]:0 tls=true :method=GET :authority=localhost :path=/some/path dst_res=po/my-pod
-end id=1:0 proxy=out src=0.0.0.1:0 dst=[ff01::1]:0 tls= grpc-status=Code(666) duration=0µs response-length=1337B
+req id=1:0 proxy=out src=0.0.0.1:0 dst=[ff01::1]:0 tls=true :method=GET :authority=localhost :path=/some/path  dst_pod=my-pod dst_tls=true 
+end id=1:0 proxy=out src=0.0.0.1:0 dst=[ff01::1]:0 tls= grpc-status=Code(666) duration=0µs response-length=1337B   
```

Comment:
it looks like there's now some trailing whitespace here, is that something we care about removing?

Reply:
These are somewhat contrived test cases because these label sets wouldn't be empty in practice.  But let me see about cleaning this up.